app/controllers/people_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# crud for people. this is essentially the directory
+class PeopleController < ApplicationController
+  def index
+    authorize Person
+    @people = Person.all
+  end
+
+  def new
+    authorize Person
+    @person = Person.new
+  end
+
+  def create
+    authorize Person
+    @person = Person.new(person_params)
+    redirect_to people_path if @person.save
+  end
+
+  def show
+    @person = Person.find(params[:id])
+    authorize @person
+  end
+
+  def edit
+    @person = Person.find(params[:id])
+    authorize @person
+  end
+
+  def update
+    @person = Person.find(person_params[:id])
+    authorize @person
```

Comment:
Similar to the above, I think the best pattern is to assign attribute, authorize, and then attempt the save. It is a bit more verbose but is generally what we want to do.